core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java

```diff
@@ -528,6 +528,10 @@ void createMeasurementEvent(
                             offset = buffer.addTimestamp(offset, entity.getLongValue());
                             break;
 
+                        case ColumnType.DATE:
```

Comment:
nit: would be nice to add a non-clitutil test for this change.

Reply:
Done